lib/source/BigQuerySourceTask.js

```diff
@@ -2,39 +2,57 @@
 
 const { SourceTask, SourceRecord } = require("kafka-connect");
 
+const DEFAULT_ID_COLUMN = 'id';
+const DEFAULT_UPDATED_AT_COLUMN = 'updated_at';
+
 class BigQuerySourceTask extends SourceTask {
 
-    start(properties, callback, parentConfig) {
+    _getQuery() {
+        return [
```

Comment:
you can write this as normal string. it was written like array in sequalize-connector because there it's sequalize :)

Reply:
it will look much better

Reply:
Yea true, okay

Reply:
I tried with it written out but it looked very messy, so I've just kept it like this but tidied it up a bit more.